lib/instance-validator.js

```diff
@@ -351,24 +369,40 @@ class InstanceValidator {
   _handleReflectedResult(field, value, promiseInspections) {
     for (const promiseInspection of promiseInspections) {
       if (promiseInspection.isRejected()) {
-        const rejection = promiseInspection.error();
-        this._pushError(true, field, rejection, value);
+        const
+          rejection = promiseInspection.error(),
+          isBuiltIn = !!rejection.validatorName;
```

Comment:
We can't have it in eslintrc because a big amount of the codebase doesn't conform, it can't be autofixed and is a lot of work to fix manually. But new code uses this style. The old style is an artifact of this:
```js
var foo = 1
  , bar = 2
  , baz = 3;
```
now with ES6 `let`/`const` there is no aestetic to it cause we have 2 different keywords now and they don't have the same length.
Having one `const` per line minimises diffs if a variable is to be removed and keeps the type of the declaration to the declaration itself.